src/network/networkd-routing-policy-rule.h

```diff
@@ -51,6 +51,7 @@ typedef struct RoutingPolicyRule {
         struct fib_rule_uid_range uid_range;
 
         int suppress_prefixlen;
+        int suppress_ifgroup;
```

Comment:
Why is this an int? If you make this `int32_t`, you can drop the casts in various places.